reference/forms/types/choice.rst

```diff
@@ -102,6 +102,11 @@ is the item value and the array value is the item's label::
 choice_list
 ~~~~~~~~~~~
 
+.. caution::
+
+    The ``choice_list`` option of ChoiceType was deprecated in Symfony 2.7.
+    You should use ``choices_as_values`` or ``choice_loader`` now.
```

Comment:
@xabbuh Honestly, I just copied this text from https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.7.md . But looking at the code example, it should probably read ``choices`` here and there as well.

The notice also mentions ``choice_loader`` but I could not find this option anywhere in the docs. Has this also been removed or is it just not documented?